lib/wit/main.py

```diff
@@ -227,8 +227,11 @@ def add_dep(ws, args) -> None:
     req_dep = dependency_from_tag(ws.root, args.pkg)
 
     cwd = Path(os.getcwd()).resolve()
-    cwd_dirname = cwd.relative_to(ws.root).parts[0]
+    if cwd == ws.root:
+        error("add-dep must be run inside of a package.\n  The cwd (which can also be set by -C) " +
+              "determines to which package a dependency will be added.")
```

Comment:
I feel the construction of this second sentence is a bit awkward, since it takes a number of detours before getting to the "main action". What if we tried flipping the order of the words like this:

> A dependency is added to the package determined by the current working directory, which can also be set by -C.

Reply:
That is better, updating then merging once CI passes 🙂